src/pages/community.html

```diff
@@ -39,23 +39,23 @@
 						<div class="list__item">
 							<div class="tile type_community">
 								<div class="tile__icon">
-									<div class="icon size_120x120 name_slack"></div>
+									<div class="icon size_120x120 name_slack"><a href="https://slackui5invite.herokuapp.com/" class="fill-div"></a></div>
```

Comment:
Having 2 links for the same URL in the same block is semantically incorrect. Can't we wrap everything in a single link tag?
Also, from UX perspective it's better when user hovers the image, the link below the image responds to the hover event.

Reply:
with the current HTML structure this is not possible - suggestion to rework it later

Reply:
Wrong statement, but okay.